src/lib.rs

```diff
@@ -232,6 +232,16 @@ mod contract {
         process_exit_reason(status, &result);
     }
 
+    #[cfg(feature = "testnet")]
+    #[no_mangle]
+    pub extern "C" fn make_it_rain() {
```

Comment:
This method makes the faucet available if you have a Near account, but what if you only have an EVM address? Should we also make the faucet available via an ethereum transaction? (I'm not sure how to do this to be honest, but it seems like a good thing for UX if we think not all users will also have Near accounts)

Reply:
Good point, for that, I'm not entirely sure. Wouldn't that need to be built into the ETH contract itself?

Reply:
One way we could do this would be to have an faucet eth contract (i.e. contract inside the EVM) and we would just have to regularly fund it via `make_it_rain`. This could probably even be done with some automation (something watches the balance of the contract and when it gets too low submits a `make_it_rain` transaction to Near).

Is this how faucets work on Ethereum testnets? Do they have a normal contract with a large balance, and when it gets low they refill it using some admin magic?

Reply:
I've written and worked with a few UTXO coin faucets, but not one with an ETH smart contract. My understanding is that they create the balance, and then you can call a method which transfers coins from the contract to the user.

Reply:
We'll have a TestNet faucet inside the EVM as well, but haven't got down the task list quite that far yet.